Update whats-happening-to-mariadb.md (#123435)
Azure/azure-content: articles/mariadb/whats-happening-to-mariadb.md

A. Unfortunately, we don't plan to support Azure Database for MariaDB beyond the sunset date of September 19, 2025. Hence, we advise that you start planning your migration as soon as possible.

> On 19 September 2025, workloads running Azure Database for MariaDB will be deleted and associated application data will be lost as announced on [Azure Database for MariaDB will be retired on 19 September 2025 – Migrate to Azure Database for MySQL Flexible Server](https://azure.microsoft.com/en-us/updates/azure-database-for-mariadb-will-be-retired-on-19-september-2025-migrate-to-azure-database-for-mysql-flexible-server/)

Reviewer: ```suggestion
On 19 September 2025, workloads that run Azure Database for MariaDB will be deleted, and associated application data will be lost. For more information, see [Azure Database for MariaDB will be retired on 19 September 2025 – Migrate to Azure Database for MySQL Flexible Server](https://azure.microsoft.com/updates/azure-database-for-mariadb-will-be-retired-on-19-september-2025-migrate-to-azure-database-for-mysql-flexible-server/)
```
Fixing link to resolve build warning, minor edits for Azure style.
